aws/hll/aws-s3-transfer-manager/src/io/path_body.rs

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::io::error::{Error, ErrorKind};
+use crate::io::stream::InputStream;
+use crate::io::stream::RawInputStream;
```

Comment:
```suggestion
use crate::io::stream::{ InputStream, RawInputStream };
```

Reply:
I feel like I expect too much out of rustfmt 🤷‍♂️